README.md

```diff
@@ -13,18 +13,14 @@ The package has two main functionalities. First, it allows for the **creation of
 
 ## Features
 ### Generating metadata from a dataset
-MetaSynth can generate metadata from any given dataset (provided as polars or pandas dataframe). 
-
-This metadata describes the structure and characteristics of the variables in the synthetic dataset (including their names, types, data types, proportion of missing values, and distribution specifications). 
-
-This metadata follows the GMF standard, [Generative Metadata Format (GMF)](https://github.com/sodascience/generative_metadata_format) and as such is designed to be easy to read. It can be exported as .JSON file allowing for manual and automatic editing, as well as easy sharing.
+MetaSynth can generate metadata from any given dataset (provided as polars or pandas dataframe) in the form of MetaFrames. MetaFrames encapsulate the structure and characteristics of each column in the original dataset (including their names, variable types, data types, proportion of missing values and distribution specifications) and serve as complete recipes for generating new synthetic data. 
```

Comment:
I would use the singular (a) MetaFrame, since you're talking about a dataset/dataframe.

Reply:
Good point, I'll correct it!